stable/cerebro/templates/ingress.yaml

```diff
@@ -1,7 +1,11 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "cerebro.fullname" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion }}
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Maybe https://github.com/helm/charts/blob/master/stable/anchore-engine/templates/ingress.yaml#L2 ? then we don't need to recollect k8s releases

Reply:
Much cleaner, thanks. I've amended the PR.